bee-message/src/output/mod.rs

```diff
@@ -201,10 +204,71 @@ impl Output {
             Self::Nft(output) => Some(output.immutable_feature_blocks()),
         }
     }
+
+    /// Checks if a sufficient storage deposit was made for the given [`Output`].
+    pub fn check_sufficient_storage_deposit(&self, config: &ByteCostConfig) -> Result<(), Error> {
+        let required_output = self.byte_cost(config);
```

Comment:
Name is a bit weird IMO

Reply:
Reopening because this was not address, just checking if it's a mistake or not. If you like this name just close again and ignore me.